requirements-dev.txt

```diff
@@ -0,0 +1 @@
+requests==2.26.0
```

Comment:
Looks like `requirements.txt` and `requirements-dev.txt` should be swapped

Reply:
Hmmm, actually looking at it, I think this `requirements.txt` pre-dates the usual divide.
I think maybe all of it needs to be in `requirements-dev.txt`.

What do you think?

Reply:
I see now, `requests` is used by the devops scripts. So yes, however `SetupShop` does require a `requirements.txt` file, but it can be empty.